tests/FunctionCallTest.php

```diff
@@ -2622,6 +2622,16 @@ function test() : void {
                     array_push();',
                 'error_message' => 'TooFewArguments',
             ],
+            'printOnlyString' => [
+                '<?php
+                    print [];',
+                'error_message' => 'InvalidArgument',
+            ],
+            'printReturns1' => [
+                '<?php
+                    (print "test") === 2;',
+                'error_message' => 'TypeDoesNotContainType',
+            ],
```

Comment:
Is this a good place for these test cases?

Reply:
Yes!